File: LuxuryBiker.Backend/LuxuryBiker.Api/Thirds/ThirdModel.cs
namespace LuxuryBiker.Api.Thirds
{
    public class ThirdModel
    {
        public string? Email { get; set; }
        public string? Identification { get; set; }
        public string? Address { get; set; }
        public bool? Active { get; set; }
        public string? CellPhone { get; set; }
        public string? Name { get; set; }
        public string? Surnames { get; set; }
        public ThirdTypeModel? Type { get; set; }
    }

    public class ThirdTypeModel
    {
        public int Id { get; set; }
        public string? Name { get; set; } = string.Empty;
    }
}

File: LuxuryBiker.ClientApp/src/domain/thirds/models/third.model.ts
export interface ThirdModel {
  id: number;
  email: string;
  identification: string;
  address: string;
  active: string;
  typeId: number;
  typeName?: string;
  cellPhone: string;
  name: string;
  surnames: string;
  fullName: string;
}

export interface TypeThirdModel {
  id: number;
  name: string;
}

export const THIRD_TYPE = {
  provider: 1,
  client: 2,
} as const;
